apache-rat/src/site/examples/default_output.txt

```diff
@@ -1,81 +1,96 @@
-
 *****************************************************
 Summary
--------
-Generated at: 2024-03-30T14:40:39+01:00
+*****************************************************
+Generated at: 2024-11-09T13:16:50Z
+
+Counters:
+
+Approved:           9    A count of approved licenses.
+Archives:           1    A count of archive files.
+Binaries:           2    A count of binary files.
+Document types:     5    A count of distinct document types.
+Generated:          1    A count of generated files.
+License categories: 5    A count of distinct license categories.
+License names:      6    A count of distinct license names.
+Notices:            2    A count of notice files.
+Standards:          8    A count of standard files.
+Unapproved:         2 (Exceeded limits)    A count of unapproved licenses.
```

Comment:
Personally I find this irritating as it breaks the readability of the report. Could we just remove the '(exceeded limits)' completely or introduce a new paragraph with 'Exceeded counters' instead? This would allow an easier parsing of the report instead of mixing valid and invalid counters. 

WDYT?